### ISSUE TITLE
Xulsword gui is in Russian

### ISSUE THREAD

**Issue by LAfricain** [JohnAustinDev/xulsword#282]
My question is simple, how to have a gui in English or in French?
Solution: Fourth entry in the menu, last entry: English.

Can I help for the French translation?

**Comment by JohnAustinDev** [comment]
Yes - I will add the French UI translation if you give me these json files with the French strings: [located here](tree/master/assets/locales/en)
